packages/runtime/container-runtime/src/index.ts

```diff
@@ -20,6 +20,13 @@ export {
 } from "./containerRuntime.js";
 export type { ICompressionRuntimeOptions } from "./compressionDefinitions.js";
 export { CompressionAlgorithms, disabledCompressionConfig } from "./compressionDefinitions.js";
+export {
+	/**
+	 * @deprecated Import from `@fluidframework/runtime-definitions/internal` instead.
```

Comment:
i'm pretty sure this doesn't work, as in the jsdoc tag is completely ignored, so doesn't show up in api report, IntelliSense, and won't trigger es lint rules. i'm not sure if there is a better pattern that actually works for this kind of thing. maybe @Josmithr  knows